src/server/routes/api/element-association/grounding-search.js

```diff
@@ -38,6 +39,10 @@ const search = opts => {
     queryOpts.organismOrdering = organismOrdering;
   }
 
+  if( namespace ){
+    queryOpts.namespace = namespace;
+  }
+
   return searchAll( queryOpts )
```

Comment:
This change makes it so that we don't send `namespace: undefined` to the query service.